README.md

```diff
@@ -28,6 +28,10 @@ If used together with Visual Studio Code and PlatformIO, starting firmware devel
 
 5. Press the upload button at the bottom left corner in VS Code.
 
+## Bootloader Support
```

Comment:
I suggest to put this section behind the flashing section (before API documentation), as the troubleshooting refers to the flashing section and now the bootloader support would be in between both.

Reply:
Agreed and updated

Reply:
Moved above the flashing section as suggested.